docs/api/permissions.rst

```diff
@@ -0,0 +1,188 @@
++-----------------+---------------------------------------------------------+
+| **record**      | The data handled by the server                          |
++-----------------+---------------------------------------------------------+
+| **group**       | A group of other :term:`principals <principal>`.        |
++-----------------+---------------------------------------------------------+
+
+There is a notion of hierarchy among all these objects:
+
+.. code-block:: text
+
+               +---------------+
+               | Buckets       |
+               +---------------+
+        +----->+ - id          +<---+
+        |      | - permissions |    |
+        |      +---------------+    |
+        |                           |
+        |                           |
+        |                           |
+        |                           |
+        |                           |
+    +---+-----------+        +------+---------+
+    | Collections   |        | Groups         |
+    +---------------+        +----------------+
+    | - id          |        |  - id          |
+    | - permissions |        |  - members     |
+    +------+--------+        |  - permissions |
+           ^                 +----------------+
+           |
+           |
+    +------+---------+
+    | Records        |
+    +----------------+
+    |  - id          |
+    |  - data        |
+    |  - permissions |
+    +----------------+
+
+
+Permissions
+===========
+
+On each of these objects, the set of permissions can be:
+
++------------+-----------------------------------------+
+| Permission | Description                             |
++============+=========================================+
+| **read**   | Any listed :term:`principal` can read   |
+|            | the object.                             |
++------------+-----------------------------------------+
+| **write**  | Any listed :term:`principal` can write  |
+|            | the object. Whoever has the permission  |
+|            | to write an object can read, update and |
+|            | delete it.                              |
++------------+-----------------------------------------+
+| **create** | Any listed :term:`principal` can create |
+|            | a new *child object*.                   |
++------------+-----------------------------------------+
+
+Permissions are associated to objects.
+
+In the case of a creation, since an object can have several kinds of children, the
+permission is prefixed (for instance ``groups:create``, ``collections:create``).
+
+The following table lists all permissions that can be associated to each kind
+of object.
+
++----------------+------------------------+----------------------------------+
+| Object         | Associated permissions | Description                      |
++================+========================+==================================+
+| Configuration  | ``buckets:create``     | Ability to create new buckets.   |
```

Comment:
The prefix "Ability to" is implied and can thus be removed from this entire list.